### PR TITLE
QuantNode Pledge

### PR DESCRIPTION
**Pull request by sshamanov**

### CODE REVIEW

File: QuantNode/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Agoric Validator Pledge
+
```
Comment (jeetraut) [inline QuantNode/README.md:2]:
```suggestion
[Quantnode Explorer Link](https://main.explorer.agoric.net/validator/agoricvaloper197cjvysnn34czsptw46vfp5swr525xhu4k5h0l)
```

**Review by jeetraut — CHANGES_REQUESTED** [review]
Add explorer link

**Review by sshamanov — COMMENTED** [review]
explorere link added

**Review by sshamanov — COMMENTED** [review]
added explorer link

**Review by jeetraut — APPROVED** [review]
Looks good

**Pull request merged**